report_aeroo/report_aeroo.py

```diff
@@ -97,15 +94,7 @@ def create_aeroo_report(
         xfunc = ExtraFunctions(cr, uid, report_xml.id, oo_parser.localcontext)
         oo_parser.localcontext.update(xfunc.functions)
 
-        if report_xml.tml_source == 'lang':
-            lang = safe_eval(report_xml.lang_eval, {'o': objects[0]})
-            template = report_xml.template_from_lang(lang)
-
-        else:
-            template = report_xml.report_sxw_content
-            if not template:
-                raise osv.except_osv(_('Error!'), _('No template found!'))
-            template = base64.decodestring(template)
+        template = report_xml.get_aeroo_report_template(objects[0])
```

Comment:
https://github.com/savoirfairelinux/aeroo_reports/pull/1/files#diff-2e806f9cdeeb7e82308dcc470a8d6787R65

because Aeroo reports are generated for one report at a time. If eventually, we need to be able to generate the report in batch, I will work on it and add unit test for this purpose.